common/src/main/java/com/wynntils/features/tooltips/ItemGuessFeature.java

```diff
@@ -91,12 +97,14 @@ private List<Component> getTooltipAddon(GearBoxItem gearBoxItem) {
 
             guesses.append(StyledText.fromString("§7: ").getComponent());
 
-            Optional<MutableComponent> itemsComponent = itemsForLevel.stream()
-                    .reduce((i, j) -> i.append(Component.literal(", ").withStyle(ChatFormatting.GRAY))
-                            .append(j));
+            MutableComponent itemsComponent = Component.empty();
+            itemsComponent.append(itemsForLevel.get(0));
```

Comment:
Won't this fail if `itemsForLevel` is empty? Maybe in the above check it should just append something like "Unknown item" or just skip to the next level

Reply:
It should never be empty, but I forgot to add the continue after the logging above.